sirepo/package_data/static/js/cloudmc.js

```diff
@@ -346,6 +346,22 @@ SIREPO.app.controller('VisualizationController', function(appState, authState, c
         return `Tally Results - ${a.tally} - ${a.score} - ${a.aspect}`;
     };
 
+    $scope.$on('settings.changed', () => {
+        for (const t of appState.models.settings.tallies) {
+            // sort and unique scores
+            const v = {};
+            const r = [];
+            for (const s of t.scores) {
+                if (! v[s.score]) {
+                    r.push(s);
+                    v[s.score] = true;
+                }
+            }
+            t.scores = r.sort((a, b) => a.score.localeCompare(b.score));
+            appState.saveQuietly('settings');
+        }
+    });
+
```

Comment:
I made a few changes to this PR. In order to get the tally score sorting right everywhere (including on the score dropdown), I sort the values before they get saved to the model.
![x](https://github.com/radiasoft/sirepo/assets/6516910/f0b8f6d9-5527-4101-8ebc-0703fa799aa8)

I found that if you had a score accidentally duplicated (like two "flux" scores in the list), openmc would fail, so I also added a uniqueness check.

- Use localeCompare() rather than sort() for strings in general so you don't get ascii sorting, and things remain alphabetical regardless of case.

@gurhar1133 give it a test and feel free to make the sorting/unique method better.

Reply:
Yeah, good call. I just made it do the sort on load since some of the examples start with unsorted tally lists.